src/generateProject/createDebugConfig.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2019 Red Hat, Inc. and others.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as fs from 'fs';
+
+import { Uri, workspace } from 'vscode';
+
+export function createDebugConfig(directory: Uri) {
+  const vscodeDir: string = directory.fsPath + '/.vscode';
+  if (!fs.existsSync(vscodeDir)) {
+    fs.mkdirSync(vscodeDir);
+  }
+
+  let tasksContent: string = tasksJson;
+  let launchContent: string = launchJson;
+  const insertSpaces: boolean = workspace.getConfiguration('editor').get('insertSpaces');
+  if (insertSpaces) {
+    const numSpaces: number = workspace.getConfiguration('editor').get('tabSize');
+    tasksContent = tasksContent.replace(/\t/g, ' '.repeat(numSpaces));
+    launchContent = launchContent.replace(/\t/g, ' '.repeat(numSpaces));
+  }
+  fs.writeFileSync(vscodeDir + '/tasks.json', tasksContent);
+  fs.writeFileSync(vscodeDir + '/launch.json', launchContent);
+}
+
+const tasksJson: string =
```

Comment:
There may be a better way to set tasks.json and launch.json through the VS Code Api. Currently investigating.

Reply:
I'd read embedded files, it's wayyy easier to maintain